### ISSUE TITLE
=, in url escaped

### ISSUE THREAD

**Issue by lightcax** [treeform/puppy#100]
https://www.**.com/list=1,2
it seems that the fetch function replace
https://www.**.com/list%3D1%2C2
resulting in a url access error, can it be unescaped?

**Comment by lightcax** [comment]
```
proc path*(url: Url): string =
  ## Returns the paths combined into a single path string.
  ## @["foo", "bar"] -> "/foo/bar"
  if url.paths.len > 0:
    for part in url.paths:
      result.add '/'
      result.add encodeURIComponent(part)
```

Is it possible to add an optional parameter not encodeURIComponent
or not escaped or to exclude =, from encodeURIComponent

**Comment by treeform** [comment]
Thank you for the bug report! You have the correct behavior, we will fix ours to match soon.

**Comment by guzba** [comment]
This is fixed in release 2.1.2. Feel free to re-open if things are still not working as expected.
